sysconfigs/daint/packages.yaml

```diff
@@ -33,6 +33,10 @@ packages:
       variants: cuda_arch=60 cosmo_target=gpu slave=daint
       version: [master]
       compiler: [pgi]
+    cosmo-c2sm:
```

Comment:
If you add those lines it should automatically use cuda_arch=60 and you shoudln't have to set it by hand, no idea why it doesn't work...